interface-definitions/include/bgp/protocol-common-config.xml.i

```diff
@@ -1431,6 +1431,12 @@
         <valueless/>
       </properties>
     </leafNode>
+    <leafNode name="route-reflector-allow-outbound-policy">
```

Comment:
I think that “allow” there is too much
“ route-reflector-outbound-policy”
@c-po what do you think?

Reply:
I would keep the allow as it explicitly tells what is going on here. The Node name is very long indeed - but I habe no other idea so far.